libs/components/layout/src/lib/modules/box/box.component.scss

```diff
@@ -29,6 +29,7 @@
         h5,
         h6 {
           margin: 0;
+          display: inline;
```

Comment:
I realize this `sky-box-header .sky-box-header` selector was already here, but I think it'd be a good idea to move to the `sky-box-header` component itself. Not sure why it's here.